install_config/install/prerequisites.adoc

```diff
@@ -797,6 +797,45 @@ See
 link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Logical_Volume_Manager_Administration/index.html[Logical
 Volume Manager Administration] for more detailed information on LVM management.
 
+[[managing-docker-container-logs]]
+*Managing Docker Container Logs*
+
+If your container's log (*_/var/lib/docker/containers/HASH/HASH-json.log_*) is
+increasing, you can use `json-file` logging driver options to manage this.
+
+For example, `--log-opt max-size` rolls over the size, and you can specify the
+`--log-opt max-file`, which is the maximum number of files that a log is rolled
+over before being discarded. You can use these options with Docker 1.8.2 or
+later.
+
+Setting in *_/etc/sysconfig/docker (max-size=1M, max-file=3)_* and restart docker
+service:
+====
+----
+OPTIONS='--insecure-registry=172.30.0.0/16 --selinux-enabled --log-opt max-size=1M --log-opt max-file=3' <1>
+----
+====
+<1> `json.log` rollover by 1M and it stored 3 files.
+
+Log output example:
+====
+----
+[root@ose3-node1 ~]# ls -lh /var/lib/docker/containers/f088349cceac173305d3e2c2e4790051799efe363842fdab5732f51f5b001fd8/
+total 2.6M
+-rw-r--r--. 1 root root 5.6K Nov 24 00:12 config.json
+-rw-r--r--. 1 root root 649K Nov 24 00:15 f088349cceac173305d3e2c2e4790051799efe363842fdab5732f51f5b001fd8-json.log
+-rw-r--r--. 1 root root 977K Nov 24 00:15 f088349cceac173305d3e2c2e4790051799efe363842fdab5732f51f5b001fd8-json.log.1
+-rw-r--r--. 1 root root 977K Nov 24 00:15 f088349cceac173305d3e2c2e4790051799efe363842fdab5732f51f5b001fd8-json.log.2
+-rw-r--r--. 1 root root 1.3K Nov 24 00:12 hostconfig.json
+drwx------. 2 root root    6 Nov 24 00:12 secrets
+----
+====
+
+The
+link:http://docs.docker.com/engine/reference/logging/overview/#the-json-file-options[Configure
+Logging Drivers] topic on *_docs.docker.com_* has more information on this
+topic.
+
```

Comment:
See our guidelines on linking: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references

Suggested edit:

The Docker site has additional information on how to http://docs.docker.com/engine/reference/logging/overview/#the-json-file-options[Configure Logging Drivers].